.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Thank you for contributing! -->
+
+## Description
+
+<!-- Please insert your description here and provide especially info about the "what" this PR is solving -->
+
+### Additional context
+
+<!-- e.g. is there anything you'd like reviewers to focus on? -->
+
+### Issue reference
+
+<!-- Please insert a link to the solved issue. If none, create one for this PR and then reference it here -->
+
+---
+
+### What is the purpose of this pull request? <!-- (put an "X" next to an item) -->
+
+- [ ] Bug fix
+- [ ] New Feature
+- [ ] Documentation update
+- [ ] Other
+
+### Before submitting the PR, please make sure you do the following
```

Comment:
This means that I will have to check all these checkboxes now when creating a new PR all the time? Is this a way to stop me or slow me down to not make that many PRs? :D

Reply:
I am not quite sure yet. But this should help us to follow our guidelines and double-check and fill in the information needed. We both know that we are submitting PRs without proper descriptions. I know I do that :-)

But if this makes no sense to you we can modify this. I also think about guidelines of healthy metrics that should be here.